modules/data.remote/inst/get_remote_data.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+get_remote_data controls GEE and AppEEARS functions to download data.
+
+Requires Python3
+
+Author: Ayush Prasad
+"""
+
+from importlib import import_module
+
+# dictionary used to map the GEE image collection id to PEcAn specific function name
+collection_dict = {
+    "LANDSAT/LC08/C01/T1_SR": "l8",
+    "COPERNICUS/S2_SR": "s2",
+    "NASA_USDA/HSL/SMAP_soil_moisture": "smap",
+  # "insert GEE collection id": "insert PEcAn specific name",          
+}
+
+
+def get_remote_data(geofile, outdir, start, end, source, collection, qc=None):
+    """
+    uses GEE and AppEEARS functions to download data
+
+    Parameters
+    ----------
+    geofile (str) -- path to the file containing the name and coordinates of ROI, currently tested with geojson. 
+    
+    outdir (str) -- path to the directory where the output file is stored. If specified directory does not exists, it is created.
+  
+    start (str) -- starting date of the data request in the form YYYY-MM-DD
+    
+    end (str) -- ending date areaof the data request in the form YYYY-MM-DD
+
+    source (str) -- source from where data is to be downloaded
+
+    collection (str) -- dataset ID
+
+    qc (float) -- quality control parameter
+
+    Returns
+    -------
+    Nothing:
+            output netCDF is saved in the specified directory.
+    """
+    if source == "gee":
+        try:
+            # get collection id from the dictionary
+            collection = collection_dict[collection]
+        except KeyError:
+            print("Requested image collection is not available")
```

Comment:
could we have a more informative message that says something along these lines "Please check if the collection name you requested is one of these and spelled correctly [prints collection_dict first column]. If not, you need to implement a corresponding gee2pecan_* function and add it to the lookup table."

Reply:
Nice!